eng/common/scripts/create-tags-and-git-release.ps1

```diff
@@ -109,7 +109,7 @@ function ParseMavenPackage($pkg, $workingDirectory) {
     return $null
   }
 
-  $releaseNotes = &"${PSScriptRoot}/../../eng/common/Extract-ReleaseNotes.ps1" -ChangeLogLocation @(Get-ChildItem -Path $pkg.DirectoryName -Recurse -Include "$($pkg.Basename)-changelog.md")[0]
```

Comment:
This looks to be the only thing in this PR in eng/common. I'm fine with that for now but we should figure out if there is more that should go in there as well.